modules/grafana/files/dashboards/rummager_queues.json

```diff
@@ -75,22 +75,22 @@
           "grid": {
             "leftMin": 0
           },
-          "leftYAxisLabel": "message count"
+          "leftYAxisLabel": "Messages per minute"
         }
       ]
     },
     {
-      "height": "400px",
+      "height": "200px",
       "editable": true,
       "collapse": false,
       "panels": [
         {
-          "title": "Publishing queue consumer queue size (RabbitMQ)",
+          "title": "Publishing queue depth (RabbitMQ)",
```

Comment:
I think queue size is more typically used to refer to the total bytes used by a queue.
It's true that the docs refer to depth as the sum of ready and acknowledged, but in practice the unack'd are in a different data structure in rabbit.